src/components/ActionControls/DrawingControls.js

```diff
@@ -2,27 +2,38 @@
 import React from 'react';
 
 import PlainButton from 'box-react-ui/lib/components/plain-button';
-import IconClose from 'box-react-ui/lib/icons/general/IconClose';
+import IconCheck from 'box-react-ui/lib/icons/general/IconCheck';
+import IconTrash from 'box-react-ui/lib/icons/general/IconTrash';
 
 import './DrawingControls.scss';
 
 type Props = {
+    canAnnotate: boolean,
```

Comment:
If we're already rendering this component. I think we can assume that the user is able to annotate.

Reply:
removed